src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ActiveConfiguredProjectsProvider.cs

```diff
@@ -75,13 +76,19 @@ public OrderPrecedenceImportCollection<IActiveConfiguredProjectsDimensionProvide
             get;
         }
 
-        public async Task<ImmutableDictionary<string, ConfiguredProject>> GetActiveConfiguredProjectsMapAsync()
+        public async Task<ImmutableDictionary<string, ConfiguredProject>?> GetActiveConfiguredProjectsMapAsync()
         {
-            var builder = PooledDictionary<string, ConfiguredProject>.GetInstance();
+            ActiveConfiguredObjects<ConfiguredProject>? projects = await GetActiveConfiguredProjectsAsync();
 
-            ActiveConfiguredObjects<ConfiguredProject> projects = await GetActiveConfiguredProjectsAsync();
+            if (projects == null || projects.Objects.Count == 0)
+            {
+                return null;
```

Comment:
Opted to let this return `null` if `GetActiveConfiguredProjectsAsync` returns null.